File: project/hyeok-onboarding/src/main/java/com/INNER_CIRCLE_ICD4/innerCircle/survey/request/repository/SurveyRepository.java
package com.INNER_CIRCLE_ICD4.innerCircle.survey.request.repository;

import com.INNER_CIRCLE_ICD4.innerCircle.survey.request.domain.Survey;
import org.springframework.data.jpa.repository.JpaRepository;

public interface SurveyRepository extends JpaRepository<Survey, Long> {
}

File: project/sungmin-onboarding/survey-service/src/main/kotlin/com/okdori/surveyservice/repository/SurveyRepository.kt
package com.okdori.surveyservice.repository

import com.okdori.surveyservice.domain.Survey
import org.springframework.data.jpa.repository.JpaRepository

/**
 * packageName    : com.okdori.surveyservice.repository
 * fileName       : SurveyRepository
 * author         : okdori
 * date           : 2025. 6. 16.
 * description    :
 */
interface SurveyRepository: JpaRepository<Survey, String> {
    fun existsBySurveyName(surveyName: String): Boolean
}
